doc/source/management/data_at_rest_encryption.rst

```diff
@@ -175,27 +175,28 @@ credentials stored in the credentials file. Location of this file is specified
 in by :variable:`keyring_vault_config`. On successful initialization it
 retrieves keys signatures and stores them inside an in-memory hash map.
 
-Configuration file should contain following information:
+Configuration file should contain the following information:
 
 * ``vault_url`` - the address of the server where Vault is running. It can be a
-  named address – like in example or ip address. The important part is that
-  it should start with ``https://``
+  named address, like one in the following example, or just an IP address. The
+  important part is that it should begin with ``https://``.
 
 * ``secret_mount_point`` - the name of the mount point where ``keyring_vault``
-  will store keys
-
-* ``token`` - the token that ``keyring_vault`` should use when connecting to
-  the Vault. When ``keyring_vault`` is to used only for transparent data
-  encryption and not for ``keyring_udf`` plugin, the token allows for storing
-  new keys in secret mount point. However if ``keyring_udf`` plugin is used
-  with ``keyring_vault`` plugin this token also allows removing keys from
-  vault. This is due to the fact that ``keyring_udf`` also supports
-  ``keyring_key_remove`` operation.
-
-* ``vault_ca [optional]`` - this variable needs to be specified only when
+  will store keys.
+
+* ``token`` - a token generated by the Vault server, which ``keyring_vault``
+  will further use when connecting to the Vault. This token can be used to
+  store new keys in a secret mount point (this the only way to apply it, when
+  ``keyring_vault`` is used only for transparent data encryption, and not for
+  ``keyring_udf`` plugin). If ``keyring_udf`` plugin is combined with 
+  ``keyring_vault`` this token also allows to remove keys from the Vault, as
```

Comment:
this token also -> this token needs to allow removing . Something like this removing is an attribute of a token